src/components/GetUser.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import Typography from '@material-ui/core/Typography';
+import User from './User';
+import RegisteredUser from './RegisteredUser';
+import GetUserForm from './GetUserForm';
+import RegisterUserForm from './RegisterUserForm';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  text: {
+    marginTop: 20,
+  },
+};
+
+class GetUser extends Component {
+  constructor (props) {
```

Comment:
Sorry missed this as well. The whole constructor can go. Last thing I promise :D

Reply:
Np, happy to iterate as much as needed   :)

Reply:
(done BTW)